app/routers/event.py

```diff
@@ -24,7 +24,7 @@
 from app.internal import comment as cmt
 from app.internal.emotion import get_emotion
 from app.internal.privacy import PrivacyKinds
-from app.internal.utils import create_model, get_current_user
+from app.internal.utils import create_model, save, get_current_user
```

Comment:
git precommit hooks to order this

Reply:
Still not ordered ^^"

Reply:
fixed my use of hooks- it's ordered now 👯